src/pad.ts

```diff
@@ -44,40 +38,40 @@ UI.pad.lightColorHash = function (author) {
 //
 //  This is more general tham the pad.
 //
-UI.pad.renderPartipants = function (dom, table, padDoc, subject, me, options) {
+export function renderPartipants (dom, table, padDoc, subject, me, options) {
```

Comment:
padDoc and me are unused.  I didn't want to delete without asking

Reply:
Removing them would mess up the positional arguments of this function.
But you can rename them to `unused1` and `unused2` and mention in the typedoc comment that these are unused positional arguments, left in place for backwards compatibility.

Reply:
cool.  will do :).